### PR TITLE
Mepts 649 apss b1 indicator

### PR DESCRIPTION
**Pull request by cfaife**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/EriDSDCohortQueries.java [hunk 1]
```diff
@@ -447,8 +446,18 @@ public CohortDefinition getPatientsWithViralLoadLessThan1000Within12Months() {
             + "                OR "
             + "                (o.concept_id=${hivViralLoadQualitative} AND o.value_coded IN (${beyondDetectableLimit},${undetectableViralLoad},${lessThan10Copies},${lessThan20Copies},${lessThan40Copies},${lessThan400Copies},${lessThan839Copies})) "
             + "            )  "
-            + "        AND e.location_id= :location  "
-       
+            + "        AND e.location_id= :location "
```
Comment (ningosi) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/EriDSDCohortQueries.java]: Are these changes related to the work outline in this ticket? this looks like DSD report

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupAPSSResumoTrimestralReport.java [hunk 2]
```diff
@@ -74,7 +74,9 @@ public ReportDefinition constructReportDefinition() {
     rd.addParameters(APSSResumoTrimestralDataSetDefinition.getParameters());
     rd.addDataSetDefinition("HF", mapStraightThrough(new LocationDataSetDefinition()));
     rd.addDataSetDefinition(
-        "R", mapStraightThrough(APSSResumoTrimestralDataSetDefinition.constructAPSSResumoTrimestralDataset()));
+        "R",
```
Comment (ningosi) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupAPSSResumoTrimestralReport.java]: I think we have search key for the other reports, can we have something that is related to the dataset we are looking at?

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/APSSResumoTrimestralCohortQueries.java [hunk 3]
```diff
@@ -16,10 +16,13 @@
 
 import static org.openmrs.module.eptsreports.reporting.library.queries.ResumoMensalQueries.*;
```
Comment (ningosi) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/APSSResumoTrimestralCohortQueries.java]: lets have individual imports as opposed to entire package imports

**Review by ningosi — COMMENTED** [review]
Have a look

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/EriDSDCohortQueries.java (hunk 1 above)
Comment (cfaife) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/EriDSDCohortQueries.java]: it might be because of the  code format  `mvn clean install`.

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupAPSSResumoTrimestralReport.java (hunk 2 above)
Comment (cfaife) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupAPSSResumoTrimestralReport.java]: what about changing from "R" to "APSS" key, does it  sound good?

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/APSSResumoTrimestralCohortQueries.java (hunk 3 above)
Comment (cfaife) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/APSSResumoTrimestralCohortQueries.java]: done

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupAPSSResumoTrimestralReport.java (hunk 2 above)
Comment (ningosi) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupAPSSResumoTrimestralReport.java]: Yes, that is more descriptive

Comment (cfaife) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupAPSSResumoTrimestralReport.java]: ok let  me change it

Comment (cfaife) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupAPSSResumoTrimestralReport.java]: done

**Review by ningosi — APPROVED** [review]
Aproved and ready to merge
